tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/ProtonTest.java

```diff
@@ -1583,6 +1583,45 @@ public void testSimpleText() throws Throwable {
    }
 
    @Test
+   public void testTimedOutWaitingForWriteLogOnConsumer() throws Throwable {
```

Comment:
test with the exact same name is already on line 534; one of them should be renamed